### DISCUSSION TITLE
ads1115 - output values divided by two

### DISCUSSION THREAD

**Discussion by pirracas77** [arendst/Tasmota#12463] [Q&A] [answered]
Hello,
I'm trying to understand how this device works.

I have connected just to test A1 to 3.3v . Unfortunately the value showed seems to be divided by two.

Is there any additional configuration that I missed?

![image](https://user-images.githubusercontent.com/18216098/123119349-bafd6c80-d443-11eb-8344-c68619c35be0.png)

![image](https://user-images.githubusercontent.com/18216098/123119416-cf416980-d443-11eb-9723-70f6a7cd9160.png)

**Comment by barbudor** [comment, marked as answer]
6,144 * 17703 / 32767 = 3,319
Sounds correct to me
Have you read the datasheet ?
Read again 9.5.4

**Reply by pirracas77** [reply]
> 6,144 * 17703 / 32767 = 3,319
> Sounds correct to me
> Have you read the datasheet ?
> Read again 9.5.4

Oh, thanks.

I saw this guy that gets direct values without applying any formula. (minunte 11:00)

https://www.youtube.com/watch?v=VixBNNKykIg&t=700s

**Reply by barbudor** [reply]
No
He gets direct value from the ADS -32678..32767 (16 bit signed as described in the datasheet)
![image](https://user-images.githubusercontent.com/2996042/123257078-fce4ec00-d4f1-11eb-8cef-66516be55085.png)

**Reply by barbudor** [reply]
While you cant' have other values in the Web GUI, you can use Rules to capture the values and applying scaling to then republish with a proper conversion 
https://tasmota.github.io/docs/Rules/
https://tasmota.github.io/docs/Commands/#scale
